spec/lib/oauth/token_request_spec.rb

```diff
@@ -1,6 +1,12 @@
 require 'spec_helper_integration'
 
 module Doorkeeper::OAuth
+  describe ClientCredentialsRequest do
+    subject do
+      # TODO: Do we need something here too?
```

Comment:
I don't know if there are any testing conventions on the project regarding "side" checking. This file and ```spec/lib/oauth/client_credentials_request_spec.rb``` seem relevant although they would test the same thing as on the file above. Should I keep only the above test?